bin/nanocld/src/services/process/stats.rs

```diff
@@ -63,3 +63,43 @@ pub async fn stats_processes(
       ),
   )
 }
+
+/// Get stats of all processes of key name
+#[cfg_attr(feature = "dev", utoipa::path(
+  get,
+  tag = "Processes",
+  path = "/processes/{name}/stats",
+  params(
+    ("name" = String, Path, description = "Name of the process", example = "deploy-example")
+  ),
+  responses(
+    (status = 200, description = "Process stats", content_type = "application/vdn.nanocl.raw-stream", body = ProcessStats),
+    (status = 404, description = "Process does not exist", body = crate::services::openapi::ApiError),
+  )
+))]
+#[web::get("/processes/{name}/stats")]
+pub async fn processes_stats_by_name(
```

Comment:
name it `process` same for the entrypoint because it handle only one process

Reply:
> name it `process` same for the entrypoint because it handle only one process

Fixed it